js/src/pages/price-benchmark/faq-link.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { recordGlaEvent } from '~/utils/tracks';
+
+const URL = 'https://woocommerce.com/document/google-for-woocommerce/faq/';
+
+const FaqLink = () => {
+	return (
+		<ExternalLink
+			onClick={ () => {
```

Comment:
@ankitrox Let’s avoid using anonymous functions, as they get redefined on every re-render in React, which can impact performance.